monkey/infection_monkey/network/zerologon_fingerprint.py

```diff
@@ -12,7 +12,7 @@
 LOG = logging.getLogger(__name__)
 
 
-class WindowsServerFinger(HostFinger):
+class ZerologonFinger(HostFinger):
```

Comment:
Why this name?

Reply:
It only conveys whether the machine is vulnerable to Zerologon, nothing else